.gitignore

```diff
@@ -1,3 +1,4 @@
 ps*.exe
 config.ahk
 custom.ahk
+config.ahk
```

Comment:
The `config.ahk` is already ignored in line 2, doesn't hurt but it shouldn't be necessary, did your git client not ignore that file?

Reply:
Yeah, prob a noob error on that lol, at first I removed the config.ahk line from there while I was doing testing trying to see what needed to be changed and such and when I was done I added it back but I reverted it or something so I didn't realize it was there and it wanted me to add another so I did, I don't know exactly but once I noticed it I wasn't sure how to fix it so I just left it, plus I assumed that if you did want to implement this stuff you would have a better way of doing it so wasn't expecting it to be merged exactly but just wanting to let you know what changed and how I fixed it basically for if you or anyone else wanted to know so it can keep being used in the meantime :)